src/cli_util.rs

```diff
@@ -580,15 +580,20 @@ impl WorkspaceCommandHelper {
     }
 
     pub fn base_ignores(&self) -> Arc<GitIgnoreFile> {
+        fn xdg_config_home() -> Result<String, VarError> {
+            if let Ok(x) = std::env::var("XDG_CONFIG_HOME") {
+                if !x.is_empty() {
+                    return Ok(x);
+                }
+            }
+            std::env::var("HOME").map(|x| format!("{x}/.config"))
```

Comment:
The function name is a bit misleading since it can return a non-xdg directory. Just drop the `xdg_` bit?

Reply:
This function implements precisely the semantics laid out in [the XDG spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), so I think the name is apt.

Reply:
Ah, I see